code/game/antagonist/outsider/vox.dm

```diff
@@ -201,7 +201,7 @@ GLOBAL_LIST_EMPTY(vox_artifact_spawners)
 		SPAN_NOTICE("\The [user] starts tinkering with [src.name]."),
 		SPAN_NOTICE("You start to analyze \the [src.name]."),
 	)
-	var/obj/item/organ/internal/voxstack/stack = user.internal_organs_by_name[BP_STACK]
+	var/obj/item/organ/internal/stack/vox = user.internal_organs_by_name[BP_STACK]
```

Comment:
The vox stacks are a different item, this will need to be changed. I'll hunt down the correct path to make sure in the morning

Reply:
var/obj/item/organ/internal/voxstack/stack = user.internal_organs_by_name[BP_STACK]
Looks like it stays the same	
	
https://github.com/Baystation12/Baystation12/blob/dev/code/game/antagonist/outsider/vox.dm

Reply:
We have them as a different item because of some of the changes made to keep cloning. I've fixed this locally, I'll push it when I get the chance.

Reply:
ah okay